src/wing/CTiglWingBuilder.cpp

```diff
@@ -39,22 +39,50 @@
 #include <Geom_Plane.hxx>
 
 #include <cassert>
+#include <algorithm>
 
 #define NO_EXPLICIT_TE_MODELING
 
+namespace
+{
+template <typename ForwardIter, typename Compare>
+size_t find_index(ForwardIter begin, ForwardIter end, Compare comp)
+{
+    const auto it = std::find_if(begin, end, comp);
+    if (it != end) {
+        return std::distance(begin, it);
+    }
+    else {
+        return std::distance(begin, end);
+    }
+}
+
+template <typename ArrayLike, typename ValueType>
+bool contains(const ArrayLike& array, ValueType val)
```

Comment:
I know this is an anonymous helper function, so feel free to ignore this comment, but the tolerance could be a parameter with a default value. Also, we could add this to the `tiglcommonfunctions`?

Reply:
I was already thinking the same. So lets move it to commonfunctions using a tolerance value.